packages/daimo-api/src/server/pushNotifier.ts

```diff
@@ -335,6 +337,8 @@ export class PushNotifier {
 
     const dollars = amountToDollars(swap[0].toAmount);
 
+    if (Number(dollars) < 0.5) return []; // Don't notify on dust, still shows in app.
```

Comment:
recommend `DUST_DOLLARS`, can set to 1 probably

Reply:
done